test/test.coffee

```diff
@@ -1,27 +1,26 @@
+'use strict'
 {expect} = require 'chai'
 Plugin = require '../'
 
 describe 'Plugin', ->
   plugin = null
 
-  beforeEach -> plugin = new Plugin
+  beforeEach -> plugin = new Plugin plugins: {}
 
   it 'should be an object', ->
-    expect(plugin).to.be.ok
+    expect(plugin).to.be.an 'object'
 
   it 'should have #optimize method', ->
-    expect(plugin.optimize).to.be.an.instanceOf Function
+    expect(plugin).to.respondTo 'optimize'
 
-  it 'should compile correctly', (done) ->
+  it 'should compile correctly', ->
     content = 'const x = 1 + 2;'
     expected = 'var x=3;'
     plugin.optimize(data: content)
-      .then (data) ->
-        expect(data).to.eql(data: expected)
-      , done()
```

Comment:
Please note `done()` call here: it is called right away, making the test "useless".